providers/base/units/wwan/test-plan.pxu

```diff
@@ -46,8 +46,6 @@ _name: Manual wwan tests
 _description: Manual wwan tests for Snappy Ubuntu Core devices
 include:
     wwan/detect-manual
-    wwan/gsm-connection-manual
-    wwan/check-sim-present-manual
     wwan/scan-networks-manual
     wwan/gsm-connection-interrupted-manual
```

Comment:
Both of `wwan/scan-networks-manual` and `wwan/gsm-connection-interrupted-manual` still depend on removed job `wwan/check-sim-present-manual`